jar-connector/src/main/java/com/sixsq/slipstream/connector/Collector.java

```diff
@@ -358,7 +347,7 @@ private static void updateDbVmsWithCloudVms(VmsClassifier classifier, EntityMana
 			if (merge) {
 				logger.info("updateDbVmsWithCloudVms: Updating db VM: id=" + dbVm.getInstanceId() + ", state=" + dbVm.getState());
 				em.merge(dbVm);
-				VirtualMachineHandler.updateVM(dbVm);
+				VirtualMachineHandler.handleVM(dbVm);
```

Comment:
@elegoff I think you should extract this line out of the `if`. 
This will allow to copy all existing VMs to the new VirtualMachines resource.